app/views/content/help-and-advice.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Help and advice
+navigation: 50
+---
+
+We’re here to answer your questions and provide free advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.
+
+##Come to an event
+
+Our events are a great way to meet people and talk about any concerns or questions you have about becoming a teacher.
+
+###Train to Teach
+
+Train to Teach events are free to attend and you'll be able to:
+
+- put your questions to expert advisers, teachers and training providers
+- chat with current teachers
+- find out what a career in teaching is really like
+- watch presentations which provide detailed guidance on how to get into teaching, the application process and funding your training
+
+
+###Online question and answer sessions
+
+There are online question and answer sessions where you can ask a panel of specialists the questions that matter to you. These sessions cover a range of different topics so you can ask specific questions and receive advice tailored to your particular circumstances.
+
+###School and university events
+
+You could also attend one of the School and University events, either online or in person, and hear directly from teacher training providers about the courses they offer.
+
+Search for an event near you (THIS IS A BUTTON)
+
+##Get a teacher training adviser
+
+Teacher training advisers provide free, one-to-one support wherever you are in your journey into teaching. You can phone, text or email them. They are experienced teachers and able to answer any questions you have. You will have your own dedicated adviser to support you at every step as you make important decisions about your future.
+
+They can help you to:
+
+- get classroom experience
+- find out what teaching is really like
+- understand more about teacher training courses
+- understand the funding and support available to you
+- find the right courses to apply for
+- make a successful application
+- prepare for your interview
+
+Get your adviser (THIS IS A BUTTON)
+
+##Join our mailing list
```

Comment:
I'm really not keen on calling this a 'mailing list' anywhere on GIT - it's how we talk about it internally and sounds really dull :-)

And 'join' just makes it sound like everyone gets the same stuff!

Reply:
Yes, it's internal language and I haven't really heard users mention a mailing list.
We currently use 'register for updates' but the terminology that resonates with users is probably something to test.

Reply:
I don't think 'updates' lands particularly well either - sounds like it is updates about teaching/DfE rather than personalised guidance

The onboarding journeys are getting more personalised now (eg re subject) so I think we need to stress this benefit